cmd/appliance/extract_logs.go

```diff
@@ -0,0 +1,185 @@
+package appliance
+
+import (
+	"archive/zip"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/appgate/sdpctl/pkg/cmdappliance"
+	"github.com/appgate/sdpctl/pkg/docs"
+	"github.com/appgate/sdpctl/pkg/factory"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/appgate/journaldreader/journaldreader"
+)
+
+type logextractOpts struct {
+	cmdappliance.AppliancCmdOpts
+	Path string
+}
+
+func NewExtractLogsCmd(f *factory.Factory) *cobra.Command {
+	aopts := cmdappliance.AppliancCmdOpts{
+		Appliance: f.Appliance,
+		Config:    f.Config,
+		CanPrompt: f.CanPrompt(),
+	}
+
+	opts := logextractOpts{
+		aopts,
+		"",
+	}
+	cmd := &cobra.Command{
+		Use:     "extract-logs",
+		Short:   docs.ApplianceLogsDoc.Short,
+		Long:    docs.ApplianceLogsDoc.Short,
```

Comment:
Some proper documentation would be nice, unless it's supposed to replace the current `appliance logs` command.